docs/content/en/docs/reference/packagespec/emissary/v0.12.1.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This file name is the wrong version

Reply:
Fixed with a13f307bb95b79204403aab4677946d793a5c9a2